### PR TITLE
Fixed grammatical (article) issue.

### PR DESCRIPTION
**Pull request by jignesh-baldha**
## This PR is a:
- [ ] New topic
- [x] Content fix or rewrite
- [ ] Bug fix or improvement
 

## Summary
Fixed grammatical (article) issue.
When this pull request is merged, it will removed **a the** article issue.

### CODE REVIEW

File: guides/v2.1/get-started/order-tutorial/order-create-quote.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Magento identifies three types of users that can create a shopping cart:
 * A guest user. These users could be customers who haven't logged in yet, or they could be users who have no intention of creating an account. An anonymous user's cart is called a guest cart.
 
 ### Create a cart for a logged-in customer {#create-cart}
-This tutorial manages a the cart of a logged-in customer. Unless otherwise noted, all calls must specify customer's token `q0u66k8h42yaevtchv09uyy3y9gaj2ap` in the header.
+This tutorial manages a cart of the logged-in customer. Unless otherwise noted, all calls must specify customer's token `q0u66k8h42yaevtchv09uyy3y9gaj2ap` in the header.
```
Comment (keharper) [inline guides/v2.1/get-started/order-tutorial/order-create-quote.md]: The sentence should be "This tutorial manages the cart of a logged-in customer."

File: guides/m1x/magefordev/mage-for-dev-5.html [hunk 2]
```diff
@@ -184,7 +184,7 @@ <h3>The Global Config and Creating The Model</h3>
 &lt;/global>
 </pre>
 
-<p>The outer &lt;weblog /&gt; tag is your Group Name, which should match your module name.  &lt;class /&gt; is the BASE name all Models in the weblog group will have, also calles <strong>Class Prefix</strong>. The &lt;resourceModel /&gt; tag indicates which Resource Model that weblog group Models should use.  There's more on this below, but for now be content to know it's your Group Name, followed by a the literal string "resource".</p>
+<p>The outer &lt;weblog /&gt; tag is your Group Name, which should match your module name.  &lt;class /&gt; is the BASE name all Models in the weblog group will have, also calles <strong>Class Prefix</strong>. The &lt;resourceModel /&gt; tag indicates which Resource Model that weblog group Models should use.  There's more on this below, but for now be content to know it's your Group Name, followed by the literal string "resource".</p>
```
Comment (shrielenee) [inline guides/m1x/magefordev/mage-for-dev-5.html]: Revise "calles" to "called".

Revise last sentence here as shown (or similar): "We talk more about this later on this page. For now, remember your Group Name and the literal string "resource".

File: guides/m1x/magefordev/mage-for-dev-5.html [hunk 3]
```diff
@@ -184,7 +184,7 @@ <h3>The Global Config and Creating The Model</h3>
...
 
 <p>So, we're not done yet, but let's see what happens if we clear our Magento cache and attempt to instantiate a blogpost Model.  In your testModelAction method, use the following code</p>
```
Comment (shrielenee) [inline guides/m1x/magefordev/mage-for-dev-5.html]: Remove double spacing after "Model."

Comment (shrielenee) [inline guides/m1x/magefordev/mage-for-dev-5.html]: Remove double spacing after "Model."

Comment (shrielenee) [reply guides/m1x/magefordev/mage-for-dev-5.html]: testModelAction should be encased in code formatting, like ` `testModelAction` `

Comment (shrielenee) [reply guides/m1x/magefordev/mage-for-dev-5.html]: Please ensure all terms that require code formatting are formatted correctly (with back ticks surround them), like <weblog />, config.xml, <resourceModel />, etc.

**Comment by shrielenee** [comment]
Also, @jignesh-baldha - please ensure you have made changes to 2.2 and 2.3 versions of the second page in your PR to match this v2.1 change.

File: guides/v2.1/get-started/order-tutorial/order-create-quote.md (hunk 1 above)
Comment (jignesh-baldha) [reply guides/v2.1/get-started/order-tutorial/order-create-quote.md]: @keharper I've changed the sentence.

File: guides/m1x/magefordev/mage-for-dev-5.html (hunk 2 above)
Comment (jignesh-baldha) [reply guides/m1x/magefordev/mage-for-dev-5.html]: @shrielenee I've updated mage-for-dev-5.html, please check now.

File: guides/m1x/magefordev/mage-for-dev-5.html (hunk 3 above)
Comment (jignesh-baldha) [reply guides/m1x/magefordev/mage-for-dev-5.html]: @shrielenee Removed unnecessary spaces.

Comment (jignesh-baldha) [reply guides/m1x/magefordev/mage-for-dev-5.html]: @shrielenee Added backticks.

**Comment by jignesh-baldha** [comment]
@shrielenee _v2.1/get-started/order-tutorial/order-create-quote.md_ is used in 2.2 and 2.3 versions so doesn't need to do changes for 2.2 and 2.3 versions.

**Pull request merged**